docs/02-connect/01-get_a_wallet.md

```diff
@@ -4,14 +4,398 @@ title: Get a Wallet
+
+Before sending a transaction, make sure that the appropriate network for recipient address is set in your Cobalt wallet.
+
+To send ZEN, perform the following steps:
+
+1. In the **Assets** page, click on the **Asset row**. 
+
+<p>
+<img src="/img/docs/get-started/gobie-cobalteon-assets.png" alt="Cobalt SendZen Assets" width="300" height="200" />
+</p>
+
+2. The **Send ZEN** page appears. Enter transaction details in the following fields:
+
+* **Send to** - Enter the wallet address you want to send to,
+* **Amount (ZEN)** - Enter the amount of ZEN you want to send. You can send an equal or less amount displayed in your available balance.  
+* **Enter your password** - Enter your wallet’s password, which acts as a signature for this transaction.
+
+3. Click **Next**.
+
+<p>
+<img src="/img/docs/get-started/gobie-cobalteon-send-zen.png" alt="Cobalt SendZen" width="300" height="200" />
+</p>
+
+4. The **Send ZEN** page appears. Review the address and estimated fee. Click **Send**.
+
+<p>
+<img src="/img/docs/get-started/gobie-cobalteon-estimated-fee.png" alt="Cobalt SendZ GasFee" width="300" height="200" />
+</p>
+
+5. Review the information and click **Send**. The **Transaction Submitted** page appears. 
+
+<p>
+<img src="/img/docs/get-started/gobie-cobalteon-transact-submitted.png" alt="Cobalt SendZ CheckMark" width="300" height="200" />
+</p>
+
+6. Optionally, verify that the send ZEN transaction is successful. Click **Link in the Explorer** field. The **EON Explorer** appears. It displays the recent Send transaction history.
+
+![cobalt-send-zen-explorer](/img/docs/get-started/gobie-cobalteon-explorer-transdetails.png)
+
+### Send ERC-20 Tokens
+
+The Cobalt wallet allows sending standard ERC20 tokens to an Ethereum address. 
+
+In this example, the LINK token is used for illustrating the send transaction. Make sure you have sufficient gas fees for the transaction. In this case, MATIC is the native currency of the Polygon network.
+
+To send ERC20 tokens, perform the following steps:
+
+1. In the **Assets** page, click on the **Asset row**
+
+<p>
+<img src="/img/docs/get-started/cobalteon-send-erc20.png" alt="Cobalt Send ERC20" width="300" height="200" />
+</p>
+
+
+2. The **Send LINK Tokens** page appears. Enter transaction details in the following fields:
+
+* **Send to** - Enter the wallet address you want to send to,
+* **Amount (LINK)** - Enter the amount of LINK you want to send. You can send an equal or less amount displayed in your available balance.  
+* **Enter your password** - Enter your wallet’s password, which acts as a signature for this transaction.
+
+<p>
+<img src="/img/docs/get-started/cobalteon-erc20-link.png" alt="Cobalt Send ERC20 Link" width="300" height="200" />
+</p>
+
+3. Click **Next**. The **Send Link Tokens** page displays the token amount that is specified, the recipient address, and estimated gas fee for the send transaction. 

```

Comment:
to be consistent with the image asset like the following:

<img width="682" alt="Screenshot 2023-04-21 at 1 15 10 PM" src="https://user-images.githubusercontent.com/5024180/233696292-8c139f75-0f5a-47b1-ae18-ad863e90054f.png">

should we rename this to **Send LINK Tokens**?

```suggestion
3. Click **Next**. The **Send LINK Tokens** page displays the token amount that is specified, the recipient address, and estimated gas fee for the send transaction. 

```